streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2675,11 +2700,6 @@ public void shouldSupportTableTableJoinWithKStreamToKTable() {
                 "    Sink: KSTREAM-SINK-0000000010 (topic: output)\n" +
                 "      <-- KTABLE-TOSTREAM-0000000009\n\n"));
 
-        final Collection<Set<String>> copartitionGroups =
-            TopologyWrapper.getInternalTopologyBuilder(topology).copartitionGroups();
-
-        assertEquals(1, copartitionGroups.size());
```

Comment:
This check is redundant, because we know there is only one sub-topology (ie, copartitionGroup) from the `topologyDescription` check above.